pennylane_lightning_gpu/_serialize.py

```diff
@@ -129,11 +140,13 @@ def _serialize_ops(
         use_csingle (bool): whether to use np.complex64 instead of np.complex128
 
     Returns:
-        Tuple[list, list, list, list, list]: A serialization of the operations, containing a list
-        of operation names, a list of operation parameters, a list of observable wires, a list of
-        inverses, and a list of matrices for the operations that do not have a dedicated kernel.
+        Tuple[list, list, list, list, list, list]: A serialization of the operations, containing a list
+        of operation names, a list of operation parameters, a list of hyper-parameters for each operation,
+        a list of observable wires, a list of inverses, and a list of matrices for the operations that
+        do not have a dedicated kernel.
     """
     names = []
+    hyper_params = []
```

Comment:
Could we put hyper_params after the params to make them arranged in the same order of return types?

Reply:
Nice catch! Easy fix :smile: